src/components/feedback/feedback.js

```diff
@@ -13,85 +14,129 @@ const environment =
       ? 'production'
       : 'staging'
     : undefined;
+const location = typeof window !== 'undefined' ? window.location : undefined;
 
 class Feedback extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       helpful: undefined,
       feedback: undefined,
-      feedbackSent: undefined
+      feedbackSent: undefined,
+      event: undefined
     };
     this.handleText = this.handleText.bind(this);
     this.handleYesNo = this.handleYesNo.bind(this);
-    this.submitFeedback = this.submitFeedback.bind(this);
+    this.handleSubmitFeedback = this.handleSubmitFeedback.bind(this);
```

Comment:
What do you think about converting all of these to arrow functions? Since we're already using the babel `transform-class-properties` plugin in this repo, so I think we should be covered when it builds!

Reply:
Yes! Good thinkin!